.semaphore/semaphore.yml

```diff
@@ -70,7 +70,7 @@ blocks:
             - yarn run cy:install
 
             # setup cypress.env from secrets
-            - cp ~/.config/cypress.env.json ./
+            #- cp ~/.config/cypress.env.json ./
```

Comment:
Why this line is commented?

Reply:
We don't have this file in this project. But maybe in another project, it could be needed